realm/realm-library/src/main/cpp/io_realm_SyncManager.cpp

```diff
@@ -59,24 +46,15 @@ struct AndroidClientListener : public realm::BindingCallbackThreadObserver {
     }
 } s_client_thread_listener;
 
-JNIEXPORT void JNICALL Java_io_realm_SyncManager_nativeInitializeSyncClient(JNIEnv* env, jclass)
-{
-    TR_ENTER()
-    if (sync_client) {
-        return;
-    }
-
-    try {
-        // Setup SyncManager
-        g_binding_callback_thread_observer = &s_client_thread_listener;
-
-        // Create SyncClient
-        sync::Client::Config config;
-        config.logger = &CoreLoggerBridge::shared();
-        sync_client = std::make_unique<Client>(std::move(config)); // Throws
+struct AndroidSyncLoggerFactory : public realm::SyncLoggerFactory {
+    std::unique_ptr<util::Logger> make_logger(Logger::Level level) override
+    {
+        auto logger = std::make_unique<CoreLoggerBridge>(std::string("REALM_SYNC"));
```

Comment:
I was trying to figure out how `RealmLog.setLevel()` works for these factory created loggers. From what I can tell, changing the LogLevel after they are created doesn't work currently?